modules/EvseManager/Charger.cpp

```diff
@@ -1077,14 +1078,14 @@ float Charger::ampere_to_duty_cycle(float ampere) {
     return dc;
 }
 
-bool Charger::set_max_current(float c, std::chrono::time_point<date::utc_clock> validUntil) {
+bool Charger::set_max_current(float c, std::chrono::time_point<std::chrono::steady_clock> validUntil) {
     if (c >= 0.0 and c <= CHARGER_ABSOLUTE_MAX_CURRENT) {
 
         // is it still valid?
-        if (validUntil > date::utc_clock::now()) {
+        if (validUntil > steady_clock::now()) {
```

Comment:
Inconsistently using `std::chrono::steady_clock` above, and `steady_clock` here.